feat: add authors data to site/authors (#319)
jenkins-infra/docs.jenkins.io: site/authors/krisstern.adoc

:page-layout: author
:page-author_name: Kris Stern
:page-github: krisstern
:page-authoravatar: ../../images/images/avatars/krisstern.png
:page-twitter: krisstern13
:page-linkedin: kris-stern

// descriptive text comes here

> Kris has been helping out with Jenkins' GSoC participation organization since 2022 and has volunteered to be a GSoC project mentor. She has participated in GSoC twice as a contributor/student previously in 2019 and 2020, and has been trained academically as an astrophysicist with a PhD in the discipline of observational astronomy obtained from the University of Hong Kong in 2021. Professionally, Kris works in the IT sector as a software engineer. She has work experiences in Python, C++, Java, JavaScript, TypeScript, HTML, CSS/Sass, JQuery, SQL, and has completed projects in software development in general and specifically in artificial intelligence/deep learning/computer vision, Qt programming, and web development. Kris is passionate about open-source and would like to share this passion with fellow learners. Currently, Kris is a part-time MCIT Online student at UPenn.

Reviewer: ```suggestion
Kris has been helping out with Jenkins' GSoC participation organization since 2022 and has volunteered to be a GSoC project mentor. 
They have participated in GSoC twice as a contributor/student previously in 2019 and 2020, and has been trained academically as an astrophysicist with a PhD in the discipline of observational astronomy obtained from the University of Hong Kong in 2021. 
Professionally, Kris works in industry as a software engineer. 
They have work experiences in Python, C++, Java, JavaScript, TypeScript, HTML, CSS/Sass, JQuery, SQL, and has completed projects in software development in general and specifically in artificial intelligence/deep learning/computer vision, Qt programming, and web development. 
Kris is passionate about open-source and would like to share this passion with fellow learners. 
Currently, Kris is a part-time MCIT Online student at UPenn.
```
